### PR TITLE
Update strings.xml && Add italian translate

### PR DESCRIPTION
**Pull request by Fast0n**

### CODE REVIEW

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
     <string name="icon_designer">Designer do Ícone (Famil Qasimov)</string>
 
     <string name="less_than_two">Erro! Não dá para calcular a diferença com menos de dois items.</string>
-    <string name="empty_please_select_two">Selecione dois items abaixo\npara comparar as diferenças</string>
+    <string name="empty_please_select_two">Selecione dois itens abaixo\para comparar as diferenças</string>
```
Comment (bernaferrari) [inline app/src/main/res/values-pt/strings.xml]: The "\npara" is acually to break the line (\n) before the word "para", there is nothing wrong on it.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
     <string name="icon_designer">Icon Designer (Famil Qasimov)</string>
 
     <string name="less_than_two">Error! Can\'t show the difference with less than two items.</string>
-    <string name="empty_please_select_two">Select two snapshots below\nto compare the differences</string>
+    <string name="empty_please_select_two">Select two snapshots below\to compare the differences</string>
```
Comment (bernaferrari) [inline app/src/main/res/values/strings.xml]: same as reported above, it is supposed to break the line before the to word.

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    <string name="notification_enable">Attiva notifiche</string>
+
+    <string name="reload">Ricarica</string>
+
+    <string name="error">ERRORE</string>
+    <string name="author">Autore</string>
+    <string name="icon_designer">Icon Designer (Famil Qasimov)</string>
+
+    <string name="less_than_two">Errore! Impossibile vedere la differenza con meno di due elementi.</string>
+    <string name="empty_please_select_two">Seleziona due elementi sotto\per confrontare le differenze</string>
```
Comment (bernaferrari) [inline app/src/main/res/values-it/strings.xml]: same as below.

**Comment by bernaferrari** [comment]
Grazie per l'attenzione! There was just a really small issue on \n which I reviewed/commented.

**Comment by Fast0n** [comment]
Perfetto grazie a te per l'app è perfetta!

File: app/src/main/res/values-pt/strings.xml (hunk 1 above)
Comment (Fast0n) [reply app/src/main/res/values-pt/strings.xml]: Ah ok sorry

Comment (bernaferrari) [reply app/src/main/res/values-pt/strings.xml]: In a "best case scenario" this breakline shouldn't be needed and be auto, but the cardview looks a lot better with it :stuck_out_tongue:

Comment (Fast0n) [reply app/src/main/res/values-pt/strings.xml]: That's okay 😊

**Comment by Fast0n** [comment]
I wait for new updates

**Pull request closed without merging**